src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 16px;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.searchBar {
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.searchBar:focus {
+  font-weight: bold;
+}
+
+.searchBar__small {
+  height: 42px;
+  padding-left: 33px;
+
+  font-size: 14px;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
you can add font-family on.searchBar with inherit value and remove this style rule from ::placeholder

Reply:
Done